lib/hacks/grid-utils.js

```diff
@@ -200,6 +200,13 @@ function testTrack(node) {
     return node.type === 'word' && /^\[.+\]$/.test(node.value);
 }
 
+function verifyRowSize(result) {
+    if (result.rowProps && !result.sizedRow) {
```

Comment:
I mean that you can just compare length here.
```js
if (result.areas.length > result.rows.length) result.rows.push('auto')
```
So you don't need additional properties (`rowProps` and `sizedRowz`)

Reply:
Thanks, now I finally see what you mean. I'll commit later today.